### DISCUSSION TITLE
How to set the cut-in, rated, cut-out wind speed in openfast

### DISCUSSION THREAD

**Discussion by L-Ethan** [OpenFAST/openfast#1224] [Q&A]
Hi. I want to do the calculation according to the parameters in the literature, but I don't know where the cut-in, rated and cut-out wind speeds are set. I think it's in the ServoDyn module, but I don't know which parameter? Would you please to tell me how to set this?
![image](https://user-images.githubusercontent.com/104621769/185635305-4fc4ccb9-74c8-4118-8de9-60da05b96723.png)
Thank you so much!

**Comment by jjonkman** [comment]
Dear @L-Ethan,

The cut-in, rated, and cut-out wind speeds are not input parameters in ServoDyn.  They may be parameters of a user-specified controller, but the baseline controller of the NREL 5-MW turbine does not make use of these values either (at least directly).  The values do determine which wind speeds you should simulate when running design load cases involving the rotor operating.

Best regards,
